lib/json_schemer/schema/base.rb

```diff
@@ -565,6 +581,14 @@ def validate_object(instance, &block)
             validate_instance(subinstance, &block)
           end
         end
+
+        if instance.after_validation && properties
```

Comment:
I wonder if people will expect to know if the data is valid at this point?

Reply:
I am  not sure. Probably yes.

Reply:
…but I cannot think of a use case where it is relevant if the data is valid or not.